### ISSUE TITLE
As a user, I should be able to add new files/resources from my computer using the resources side bar?

### ISSUE THREAD

**Issue by sanjayankur31** [OpenSourceBrain/OSBv2#341]
When adding more resources to workspaces, users can either add a URL or use a OSBv2 repo. Should they also be able to upload files from their local machines? They can already do this using the Jupyterlab interface, but this is not doable from the resources side bar.

So, this should ideally be consistent---either they should be able to upload files/resources from their local machines using the resources side bar, or they should not be able to do it using the jupyterlab interface either?

**Comment by filippomc** [comment]
Yes that's part of the design but haven't got prioritized 
![image](https://user-images.githubusercontent.com/46561561/151136900-5565a48f-b764-4f8b-b9da-5ceeecaccbfd.png)

**Comment by pgleeson** [comment]
Should be careful too about what exactly a "resource" is... Not every file is a resource, it's really "a file of a type that one or more OSBv2 application recognises and can be opened by that application".

This is clear cut for NWB files, the future NetPyNE format, NeuroML files (eventually will be openable directly by NetpyNE-UI), and jupyter notebooks. It could become blurry for image files say which can be happily opened by JupyterLab...

**Comment by filippomc** [comment]
Yes agree, maybe could add some test here and there. And there is no actually any limitation atm in what can be added from the resource add interface: for instance an image is added but there is no feedback to the user unless they open the workspace on jupyterlab

**Comment by pgleeson** [comment]
There **could** be an option to "show all files" in the resources tab, where all files of non recognised types are shown greyed out (option disabled by default). The others like nwb files can be clicked on and something will happen whereas the others are disabled.

**Comment by filippomc** [comment]
That would be an option but let's keep in mind that showing all files is possible but needs some thinking and work: apart of the frontend implementation, which by itself may need some redesign, the backend does not have access to the workspaces file system for how the volume assignments has been designed.
